src/main/java/kata/makers/model/Flight.java

```diff
@@ -0,0 +1,16 @@
+package kata.makers.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@AllArgsConstructor
+@Setter
+@Getter
+@ToString
```

Comment:
Do we need all these annotations?

Reply:
I think `@ToString` is just for my debugging 😅 so it is not necessary
For `@Setter` and `@Getter`, it can be removed, double checked I just use it in test

While for `@AllArgsConstructor`, I think I have keep it as we have to create a new object